.ci/updatecli/values.d/scm.yml

```diff
@@ -0,0 +1,7 @@
+scm:
+  enabled: true
+  owner: elastic
+  repository: apm-agent-nodejs
```

Comment:
I'm starting to wonder if we could utilize `$GITHUB_` env vars for this.

But in the shared policy.

Reply:
They are there, but I forced it to play safe.

```

# {{ $GitHubRepositoryList := env "GITHUB_REPOSITORY" | split "/"}}

...

{{ if or (.scm.enabled) (env "GITHUB_REPOSITORY") }}
scms:
  default:
    kind: "github"
    spec:
      # Priority set to the environment variable
      user: '{{ default $GitHubUser .scm.user}}'
      email: '{{ .scm.email }}'
      owner: '{{ default $GitHubRepositoryList._0 .scm.owner }}'
      repository: '{{ default $GitHubRepositoryList._1 .scm.repository}}'
...
```

Reply:
Ah okay. 👍